epictrack-web/src/components/shared/index.tsx

```diff
@@ -186,9 +186,9 @@ export const ETParagraph = React.forwardRef(
           <Typography
             color={color}
             sx={{
-              fontSize: "15px",
-              lineHeight: "24px",
-              letterSpacing: "-0.32px",
+              // fontSize: "15px",
+              // lineHeight: "24px",
+              // letterSpacing: "-0.32px",

```

Comment:
can we remove all together instead of commenting out?

Reply:
Sure thing

Reply:
@jadmsaadaot Updated as per your comments. Also, updated the enhancements done in detail in the description above. Please check.

Reply:
thanks for the changes, approved